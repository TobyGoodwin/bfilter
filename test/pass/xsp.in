Dull: header
Subject: interesting header
X-Spam-Probability: YES (p=1.00000)
  we can even continue this line
Date: 1st April 1968

body
