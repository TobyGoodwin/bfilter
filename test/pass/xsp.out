Dull: header
Subject: interesting header
Date: 1st April 1968

body
